lib/ModelStore.ts

```diff
@@ -2,9 +2,9 @@ import CustomError = require("./Error");
 import {Model} from "./Model";
 import {Document} from "./Document";
 
-let models: {[name: string]: Model<Document>} = {};
+let models = {};
```

Comment:
I think this should be typed somehow. The current type is `{}`. Which leads to an issue where something like `models["number"] = 1;` is allowed, when it shouldn't be.

Reply:
ok i will put it back